src/towncrier/_builder.py

```diff
@@ -224,6 +241,11 @@ def split_fragments(
                 # it's recorded.
                 content = ""
 
+            # Calculate the underline size for the name of the category.
```

Comment:
I don't think that this comment is needed.

The code is pretty clear.

```suggestion
```

Reply:
Removed